### DISCUSSION TITLE
Where is $Path?

### DISCUSSION THREAD

**Discussion by attishno1** [DrMoriarty/nativelib-cli#4] [Q&A]
Hello! Thank you for viewing!

I am on Linux Mint. Where is $Path located?

Will use this with Facebook Module.

**Comment by DrMoriarty** [comment]
Hi!
You can check your path by running this command:
```
echo $PATH
```
I have no exact recommendations to how you could change it, because it depends on shell you are using.

**Reply by attishno1** [reply]
Thank you for replying

I am on Linux Mint. I did what you asked. However, I got a number of paths. I am copy pasting them here

/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/home/attishno1/.dotnet/tools:/home/attishno1/Downloads/flutter/bin

Which one should I use?

**Reply by DrMoriarty** [reply]
Usually for custom scripts and applications used `/usr/local/bin/`
